### DISCUSSION TITLE
Issue with React hook form after React Native upgrade

### DISCUSSION THREAD

**Discussion by eruanomamud** [react-hook-form/react-hook-form#7916] [Q&A]
Hi guys,

I recently upgraded my React Native from 0.63 to 0.67 and I started getting the next error when the app is opening:
```
error: node_modules/react-hook-form/dist/index.cjs.js: name => {
          if (file.availableHelper && !file.availableHelper(name, runtimeVersion)) {
            retu...<omitted>... } could not be cloned.
```

I have been looking online with no success, and I was a couple of versions behind for React Hook Form and after updating to the latest version, I am still seeing the error.
At this point, I am not sure if this error is related to React Hook Form or React native. Also, I am seeing the error on iOS, for Android I am getting some other issues related to the upgrade, so I am not sure if I will get this same error on there too.

Thanks and I appreciate any help at all,
Ernesto

**Comment by bluebill1049** [comment]
`if (file.availableHelper && !file.availableHelper(name, runtimeVersion)) {` that doesn't looks like code from hook form. can you share your usage? could be related to file object, which probably shouldn't be a part of hook form's `defaultValues`.

**Reply by Thiago-Offredi** [reply]
Error in ./~/react-hook-form/dist/index.cjs.js
Module parse failed: /home/thiagooffredi/Front-Burguer/code-burger-front/node_modules/react-hook-form/dist/index.cjs.js Unexpected token (1:1685)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (1:1685)
 @ ./src/containers/Login/index.js 12:21-47
Ao importar ja da problema como resolve isso ajuda a gente ai por favor

**Reply by leapful** [reply]
@Thiago-Offredi 

Please open a new discussion with codesandbox url to re-produce the issue. Your issue might be not related to the problem of this current discussion.
